### PR TITLE
python3Packages.django-hcaptcha: init at 0.2.0

### PR DESCRIPTION
**Pull request by MrMebelMan**
###### Description of changes

  Add a new package.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pbsds** [comment]
Please rebase this on top of master
```
Auto-merging pkgs/top-level/python-packages.nix
CONFLICT (content): Merge conflict in pkgs/top-level/python-packages.nix
Automatic merge failed; fix conflicts and then commit the result.
```

**Comment by MrMebelMan** [comment]
> Please rebase this on top of master
> 
> ```
> Auto-merging pkgs/top-level/python-packages.nix
> CONFLICT (content): Merge conflict in pkgs/top-level/python-packages.nix
> Automatic merge failed; fix conflicts and then commit the result.
> ```

Fixed!

File: pkgs/development/python-modules/django-hcaptcha/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, django, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "django-hCaptcha";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/django-hcaptcha/default.nix:4]:
```suggestion
  pname = "django-hcaptcha";
```
we normalise the attr and pname

File: pkgs/development/python-modules/django-hcaptcha/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  version = "0.2.0";
+  disabled = isPy27;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/django-hcaptcha/default.nix:6]: Please use pythonOlder

File: pkgs/development/python-modules/django-hcaptcha/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06kcrcm24q21k6anhbc5wkhn3ky5488k09gqfan6ay691jprwldj";
+  };
+
+  # Requires hCaptcha credentials
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/django-hcaptcha/default.nix:14]: Please add a pythonImportsCheck

**Comment by mweinelt** [comment]
Superseded by 5c71b2335b8ecbb0aa18d0c2dfd1c4c4606f4985

**Pull request closed without merging**